tool/jt.rb

```diff
@@ -516,7 +516,9 @@ def build(*options)
       raw_sh "make", "-j#{cores}", chdir: "#{TRUFFLERUBY_DIR}/src/main/c"
     when nil
       mx 'sforceimports'
-      mx 'build', '--force-javac', '--warning-as-error'
+      mx 'build', '--force-javac', '--warning-as-error',
+         # show more than default 100 errors not to hide actual errors under pile of missing symbols
+         '-A-Xmaxerrs', '-A1000'
```

Comment:
Should we also have this setting in CI?
Or you always `jt build` on the command line and IntelliJ never compiles by itself?

Reply:
> Or you always jt build on the command line

Yes that's what I do. Usually `mx build run ...`.